amsclient/amsclient.go

```diff
@@ -305,3 +306,9 @@ func (c *amsClientImpl) executeSubscriptionListRequest(
 
 	return
 }
+
+func (c *amsClientImpl) IsClusterInOrganization(orgID types.OrgID, clusterID types.ClusterName) bool {
+	clusterInfo, err := c.GetSingleClusterInfoForOrganization(orgID, clusterID)
```

Comment:
+1

Reply:
Locally errcheck was fine with that, but I'll double check

Reply:
I double checked and goerrcheck doesn't have any trouble with it, as the result of the function depends on checking the `err`. Are you ok with it or do you prefer to return bool + error?

Reply:
but in case there is an error, we won't know what error, it's not even logged, that's the problem. I'm really surprised it didn't complain about unchecked errors

Reply:
It doesn't complain because the error is checked (in the return clause).

And I didn't add any log because all the possible errors are already logged inside the called function, so adding one more log will duplicate the message